utils/discord-sender.js

```diff
@@ -1,16 +1,35 @@
+// @flow
 import request from 'request-promise';
 import config from '../config.json';
 import logger from '../utils/logger';
+import {translate} from '../i18n';
 
-function formatDiscordDescription(event) {
-    return `**Quando:** ${event.eventDate}
-**Dove:** ${event.location}
-**Chiusura application:** ${event.applicationClose}`;
+function formatDiscordDescription(event: $FlowFixMe, language: string = 'en') {
+    const t = translate(language);
+    return `**${t('when', 'When:')}** ${event.eventDate}
+**${t('where', 'Where:')}** ${event.location}
+**${t('closing', 'Applications close:')}** ${event.applicationClose}`;
 }
 
-export default function postEvent(event) {
+export default function postEvent(event: $FlowFixMe) {
     logger.info('Posting event to discord: ', event);
+    const t = translate(config.language);
+    const description = formatDiscordDescription(event, config.language);
+    const content = `${t(
+        'content.open',
+        'Applications are open for',
+    )} **${event.name}**! ${t(
+        'content.closeDate',
+        'Applications will close in:',
+    )} *${event.applicationCloseRemaining}*`;
+
     if (process.env.DRY_RUN) {
+        logger.info(`DRY_RUN selected.
```

Comment:
I think we should use `logger.debug` for DRY_RUN env.

Reply:
Yeah, that's fine, I'll change this.